src/richie-front/js/components/CourseGlimpse/CourseGlimpse.spec.tsx

```diff
@@ -1,4 +1,4 @@
-import '../../testSetup.spec';
```

Comment:
`s/a utility/an utility/`? in the commit message

Reply:
I think it's `a utility` as `utility` starts with a consonant sound